demos/embeddings/deep-graph-infomax-cora.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>I don't think we need to create a new <code>FullBatchNodeGenerator</code> for each of these methods and instead reuse the one defined above (potentially after renaming it); or do you think it's clearer for users this way?</p><p> </p><p>Also, per the suggestion earlier, I think it would better to not overwrite <code>base_model</code>, because that means a user can't jump around and easily rerun cells; if they run the whole notebook and then want to go back to an earlier cell (e.g. adjust the logistic regression parameters for the first GCN), they won't be able to do it properly without recreating the right <code>base_model</code> variable. Specifically, I think we can call this <code>gat_model</code>, with <code>gcn_model</code> earlier and <code>graphsage_model</code> later. The same applies for the <code>generator</code> variables.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>